### ISSUE TITLE
Support for command-line parameters instead of interactive approach

### ISSUE THREAD

**Issue by broeser** [mghoneimy/php-graphql-oqm#4]
Thanks for this amazing project, which I've been using for about a year now. :)

In my use case the schema objects change quite frequently. The generate_schema_objects to regenerate them works fine, but putting in the GraphQL-Endpoint, Authorization info & output folder every time is not ideal for me.

I would prefer if I could specify them as command-line arguments (that way I could wrap the script into another one and just call it with the same parameters every time; Or I would put the values in env variables).

I think this functionality would be very beneficial. What do you think?

**Comment by mghoneimy** [comment]
Hello @broeser ,
I'm glad that this package has been useful to you in your work.
I agree with you, that would be a very good feature in my opinion too.
Would you like to create a pull request for it or would you rather wait for me to work on this when I have time?

**Comment by broeser** [comment]
Hi,
I don't feel confident enough to work on this yet, so I prefer to wait. Thank you!

**Comment by bbaga** [comment]
As a workaround, this could be used:
```
echo "https://my-api.host\nAuthorization\nBearer MY_TOKEN" \
| vendor/bin/generate_schema_objects
```

**Comment by mghoneimy** [comment]
addressed by #28
